jetbrains-core/src/software/aws/toolkits/jetbrains/settings/DynamicResourcesConfigurable.kt

```diff
@@ -78,6 +94,12 @@ class DynamicResourcesConfigurable : BoundConfigurable(message("aws.settings.dyn
         }
     }
 
+    private fun submitSuggestion(suggestion: String) {
+        coroutineScope.launch(getCoroutineBgContext()) {
+            TelemetryService.getInstance().sendFeedback(Sentiment.NEGATIVE, suggestion, mapOf(FEEDBACK_SOURCE to "Resource Type Suggestions"))
```

Comment:
does exception need to be handled here?

Reply:
yeah I suppose we should at least log it